app/src/main/java/com/aptoide/uploader/account/view/LoginNavigator.java

```diff
@@ -26,7 +28,35 @@ public void navigateToMyAppsView() {
   }
 
   public void navigateToCreateStoreView() {
-    navigateTo(CreateStoreFragment.newInstance());
+    fragmentManager.beginTransaction()
+        .replace(R.id.activity_main_container, CreateStoreFragment.newInstance())
+        .addToBackStack(String.valueOf(R.layout.fragment_create_store))
+        .commitAllowingStateLoss();
+  }
+
+  public void navigateToLoginFragment() {
+    navigateTo(LoginFragment.newInstance());
+  }
+
+  public void navigateToAutoLoginFragment(String name, String avatarPath) {
+    Bundle bundle = new Bundle();
+    bundle.putString("name", name);
+    bundle.putString("avatarPath", avatarPath);
+
+    Fragment fragment = AutoLoginFragment.newInstance();
```

Comment:
Because we are creating a static helper method to instantiate the fragment, we should pass the parameters (name and avatar path) to the newInstance. This way you "enforce" from the fragment side the use of those paramenters. Because if they are not passed (imagine navigating to the same fragment somewhere else) it will crash.

Reply:
Makes total sense and that way I dont need to use Bundle to pass the name and avatar from Navigator to Fragment.